### DISCUSSION TITLE
#317 PR help

### DISCUSSION THREAD

**Discussion by EbonyLouis** [Virtual-Coffee/virtualcoffee.io#390] [Q&A] [answered]
Good morning, I worked on issue #317 and my PR is allowing me to merge my changes without a review. I'm wondering if that means I made my PR incorrectly. I forked the repo, made my changes locally,created a PR and linked the issue number.

Was that the incorrect way of doing it?

**Comment by danieltott** [comment, marked as answer]
Hey Ebony!

The problem here is the base branch in your Pull Request. It is set you main on your fork, but what it needs to be to get back to the site is main on Virtual Coffee's version. You should be able to change this by hitting edit next to the PR title and then choosing the correct base from the drop downs - something like Virtual-Coffee/virtualcoffee.io/main

**Reply by EbonyLouis** [reply]
Okay thank you, I'll follow that now.

**Reply by EbonyLouis** [reply]
It wasn't giving me the correct base as an option. So I merged it into my repo and then it said I was behind the original repo. So I created a new PR there. So in short it should be correct now lol. 🤞🏾
